pkgs/by-name/dn/dns-over-https/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  nix-update-script,
+}:
+buildGoModule rec {
+  pname = "dns-over-https";
+  version = "2.3.7";
+
+  src = fetchFromGitHub {
+    owner = "m13253";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
Resolved